recipes-support/bt-6lowpan-setup/bt-6lowpan-setup_0.1.bb

```diff
@@ -0,0 +1,34 @@
+SUMMARY = "Setup BT 6LoWPAN network / modules"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+inherit allarch
+
+SRC_URI = " \
+	file://bt-6lowpan.network.in \
+	file://modules-6lowpan.conf \
+"
+
+S = "${WORKDIR}"
+
+PACKAGE_ARCH = "${MACHINE_ARCH}"
+
+# Allow build time customizations by the user
+BT_6LOWPAN_INTERFACE ?= "bt0"
+BT_6LOWPAN_NETWORK ?= ""
+
+do_compile() {
+	if [ ! -z "${BT_6LOWPAN_NETWORK}" ]; then
+		sed -e 's/@@BT_6LOWPAN_NETWORK@@/${BT_6LOWPAN_NETWORK}/' \
+			${S}/bt-6lowpan.network.in > ${S}/bt-6lowpan.network
+	fi
+}
+
+do_install() {
+	if [ -f "${S}/bt-6lowpan.network" ]; then
+		install -d ${D}${systemd_unitdir}/network
+		install -m 0644 ${S}/bt-6lowpan.network ${D}${systemd_unitdir}/network/60-bt-6lowpan.network
+	fi
+	install -d ${D}${libdir}/modules-load.d
+	install -m 0644 ${S}/modules-6lowpan.conf ${D}${libdir}/modules-load.d/6lowpan.conf
```

Comment:
Here since modules-6lowpan.conf gets provided as a file, better to use ${WORKDIR} instead (${S} works the same way but it can later mean a different thing if the recipe starts fetching sources from a git repository, for example).

Reply:
If that's the case, do you want me to change to ${WORKDIR} for the following lines in sbin-path-helper, sysctl-hang-crash-helper and sysctl-net-queue-pfifo-fast? (these are provided as files):
`install -m 0644 ${S}/path-sbin.sh ${D}${sysconfdir}/profile.d/path-sbin.sh`
`install -m 0644 ${S}/sysctl-panic.conf ${D}${libdir}/sysctl.d/60-panic.conf`
`install -m 0644 ${S}/sysctl-net-queuing.conf ${D}${libdir}/sysctl.d/90-net-queuing.conf`

Reply:
I went ahead and followed the same methodology for the above in the new patchset.